### DISCUSSION TITLE
[Windows] Convert hIcon to Avalonia Bitmap

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#15995] [Q&A]
Hi,
How can I convert hIcon to Avalonia Bitmap?

**Comment by timunie** [comment]
Maybe this can help you? https://github.com/amwx/FluentAvalonia/blob/master/src/FluentAvalonia/Interop/Helpers/HBITMAP.cs

Credits to @amwx

**Reply by workgroupengineering** [reply]
Thanks, I'll take a look at it
